### PR TITLE
Fix the CI

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

**Comment by leofeyer** [comment]
As @ausi pointed out, Monolog 2.3.0 has been released and most likely requires some adjustments on our end.

**Comment by m-vo** [comment]
> As @ausi pointed out, Monolog 2.3.0 has been released and most likely requires some adjustments on our end.

Shall we update our requirements then or add a conflict? (until adjusted)

File: core-bundle/src/Monolog/ContaoTableProcessor.php [hunk 1]
```diff
@@ -48,8 +48,6 @@ public function __construct(RequestStack $requestStack, TokenStorageInterface $t
 
     /**
      * Move the Contao context into the "extra" section.
-     *
-     * @return array<string,array<string,mixed>>
```
Comment (m-vo) [inline core-bundle/src/Monolog/ContaoTableProcessor.php:52]: What's the issue with this typehint?

Comment (ausi) [reply core-bundle/src/Monolog/ContaoTableProcessor.php:52]: The inherited one is better now.

**Comment by ausi** [comment]
> (until adjusted)

Done 🎉

**Review by m-vo — DISMISSED** [review]

File: core-bundle/tests/Monolog/ContaoTableProcessorTest.php [hunk 2]
```diff
@@ -23,24 +23,48 @@
...
+        $record = [
+            'level' => Logger::DEBUG,
+            'level_name' => 'DEBUG',
+            'channel' => '',
+            'context' => ['contao' => false],
+            'extra' => [],
+            'datetime' => new \DateTimeImmutable(),
+            'message' => '',
+        ];
+        $this->assertSame($record, $record);
```
Comment (leofeyer) [inline core-bundle/tests/Monolog/ContaoTableProcessorTest.php]:
```suggestion

        $this->assertSame($record, $record);
```

Comment (ausi) [reply core-bundle/tests/Monolog/ContaoTableProcessorTest.php]: Done in fd17c9bc31874974972f48b4cad836553628d32d

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**